tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -38,6 +38,11 @@ use version_utils 'is_sle';
 sub run_test {
     my ($self) = @_;
 
+    #Ensure that there is enough free memory for virtual network test
+    my $MEM = $self->get_free_mem();
+    record_info('Detect FREE MEM', $MEM . 'G');
+    assert_script_run("test $MEM >= 20", fail_message => "The SUT needs at least 20G FREE MEM for virtual network test");
```

Comment:
This "test $MEM >= 20" seems to not work correctly? I run locally, both 10 and 30 can pass the "test "

Reply:
great, try to use `-ge` instead of `>`  way for test command now.